### DISCUSSION TITLE
Removing termux from the playstore kills support for chromebooks that don't have developer mode

### DISCUSSION THREAD

**Discussion by fgclue** [termux/termux-app#3518] [General]
Hello. I use termux on a chromebook, with Developer Mode off.

I opened termux today and saw this warning saying that I should switch to the **F-droid** version because the Play Store version was probably gonna be removed. I tried installing F-droid, but I need to turn on Developer Mode. 

Please, if possible, don't remove the old version of Termux from the Play Store.

Thanks,
fgclue.

**Comment by twaik** [comment]
You can download APK from F-Droid even without installing it, there is `Download APK` button on application page.
For installing it you should use this:
https://www.xda-developers.com/how-sideload-apps-chromebook/#:~:text=Go%20to%20Files%20on%20your,it%20listed%20in%20your%20launcher

**Reply by fgclue** [reply]
You still need developer mode for installing APKs from places that are not the Google Play Store

**Comment by agnostic-apollo** [comment]
You should be able to use adb without enabling developer mode on newer versions. But regardless of that, app will not be updated on playstore anytime soon, if ever. You should have read the links in the message.

https://developer.android.com/topic/arc/development-environment

https://github.com/termux/termux-app#google-play-store-deprecated
